src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -705,7 +705,7 @@ export class PatientViewPageStore {
     }
 
     static getMutationId(m: any): string {

```

Comment:
(1) already fixed in integration/release branches (was introduced by #326)
(2) already fixed in integration/release branches (was introduced by #326)

I don't want to fix those again in hotfix branch :)